### PR TITLE
Raise PHP version requirement to PHP 5.5

### PR DESCRIPTION
**Pull request by simonschaufi**
Fixes https://github.com/kimai/kimai/issues/1124

### CODE REVIEW

File: documentation/installation.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: kimai
-title: Documentation for Kimai
-description: Documentation for Kimai Time-Tracking
+title: Kimai Installation
+description: Kimai Installation
```
Comment (kevinpapst) [inline documentation/installation.md]: This is going to be shown in the google search results. Could you re-phrase that a little bit ... so make it at least 100 character longs?

**Comment by kevinpapst** [comment]
Why 5.5? https://github.com/kimai/kimai/issues/1124#issuecomment-366684919 - do you have a specific reason for that?

**Comment by simonschaufi** [comment]
Just in case some internal servers still use this old php version, it's not a big security issue as the servers are just internal.

**Comment by kevinpapst** [comment]
PHP 5.5 is out-of-live. 
And I would guess that at least 50% of all Kimai installations are hosted on a public network.

**Comment by simonschaufi** [comment]
> hosted on a public network

from the users feedback, I would imagine that most installations are in intranets with self setup servers.

File: documentation/installation.md (hunk 1 above)
Comment (simonschaufi) [reply documentation/installation.md]: oh, I'm not good in SEO. Need to do some research on how to optimize the description

**Comment by kevinpapst** [comment]
fine then, lets go with 5.5 for now

File: documentation/installation.md (hunk 1 above)
Comment (kevinpapst) [reply documentation/installation.md]: 160 character is the best. Just think about the small snippet, that is displayed under link the search result link. What would yourself when you see the result? Well, it wasn't better before ;-) but we could make it even more shiny

**Pull request merged**